design/role.go

```diff
@@ -5,6 +5,45 @@ import (
 	a "github.com/goadesign/goa/design/apidsl"
 )
 
+var _ = a.Resource("roles", func() {
+	a.BasePath("/roles")
+	a.Action("list", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.GET(""),
+		)
+		a.Params(func() {
+			a.Param("resource_type", d.String, "resource type for which roles are being listed")
+		})
+		a.Description("List available roles by resource type")
+		a.Response(d.OK, rolesMedia)
+		a.Response(d.InternalServerError, JSONAPIErrors)
+		a.Response(d.BadRequest, JSONAPIErrors)
+		a.Response(d.NotFound, JSONAPIErrors)
```

Comment:
plus Unauthorized

Reply:
45f82fd12f13c96f1186470ee81d101578f91a22 thanks.

Reply:
Sure. It’s just for swagger docs.